skimage/draw/tests/test_draw.py

```diff
@@ -276,6 +298,21 @@ def test_circle_perimeter_andres():
     )
     assert_array_equal(img, img_)
 
+    img = np.zeros((8, 8), 'uint8')
+    rr, cc = circle_perimeter(3, 2.4, 2.1, method='andres')
+    img[rr, cc] = 1
+    img_ = np.array(
+        [[0, 0, 0, 0, 0, 0, 0, 0],
+         [1, 1, 1, 1, 1, 0, 0, 0],
+         [1, 0, 0, 0, 1, 0, 0, 0],
+         [1, 0, 0, 0, 1, 0, 0, 0],
+         [1, 0, 0, 0, 1, 0, 0, 0],
+         [1, 1, 1, 1, 1, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0],
+         [0, 0, 0, 0, 0, 0, 0, 0]
+         ])
```

Comment:
@bewithaman I think @vighneshbirodkar is suggesting that you make the circle large enough that a person can see it is an actual circle (because here it looks like a square)